scripts/build.py

```diff
@@ -426,25 +425,41 @@ def process_html(source_file_stream, target_file_stream, file_hashes):
         # This is because html paths are used by backend and we work with
         # paths without hash part in backend.
         if not filepath.endswith('.html'):
+            # The path to the required file in the compiled HTML is in url,
+            # while on Windows, the path here will be in Windows-Style. We need
+            # to convert the Windows-Style path to url style so the target
+            # path can be successfully replaced with the one with hash.
```

Comment:
with the one with hash --> with the hashed path

Reply:
Updated